### PR TITLE
Sync NuGet package versions.

### PR DESCRIPTION
**Pull request by AndyGerlicher**
I went through all the NuGet packages we referenced and got them in sync. Horrible manual process unfortunately. I also removed the daily feeds so it wouldn't look there since it shouldn't be needed right now.

This now builds on my machine, hopefully the CI will pass.

### CODE REVIEW

File: Samples/PortableTask/PortableTask.csproj [hunk 1]
```diff
@@ -10,6 +10,7 @@
     <AssemblyName>PortableTask</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <UseProductOutputPath>true</UseProductOutputPath>
+	<CopyNuGetImplementations>false</CopyNuGetImplementations>
```
Comment (rainersigwald) [inline Samples/PortableTask/PortableTask.csproj]: Looks suspicious.

File: Samples/TaskUsageLogger/TaskUsageLogger.csproj [hunk 2]
```diff
@@ -9,6 +9,7 @@
     <AssemblyName>TaskUsageLogger</AssemblyName>
     <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+	<CopyNuGetImplementations>false</CopyNuGetImplementations>
```
Comment (rainersigwald) [inline Samples/TaskUsageLogger/TaskUsageLogger.csproj]: Ditto

**Comment by rainersigwald** [comment]
LGTM

**Comment by AndyGerlicher** [comment]
This is failing because of registry access which is no longer available on the Windows build. @rainersigwald is working on disabling those tests.

**Pull request merged**